java/java.disco/src/org/netbeans/modules/java/disco/BundleTableModel.java

```diff
@@ -41,57 +69,37 @@ public BundleTableModel(final List<Pkg> bundles) {
     }
 
 
-    public List<Pkg> getBundles() { return bundles; }
+    public List<Pkg> getBundles() {
+        return bundles;
+    }
+
     public void setBundles(final List<Pkg> bundles) {
         this.bundles = bundles;
         this.fireTableDataChanged();
     }
 
-    public @NonNull String getColumnName(final int col) {
-        switch(col) {
-            case 0 :
-            case 1 :
-            case 2 :
-            case 3 :
-            case 4 :
-            case 5 : return columnNames[col];
-            default: throw new IllegalArgumentException("Column not found " + col);
-        }
+    @Override
+    public int getRowCount() {
```

Comment:
Minor remark: I wish the order of methods would have been kept the same. Diff would have been easier.

Reply:
should I move it back?